### PR TITLE
[Terminology] Align terminology

### PR DESCRIPTION
**Pull request by goldmermaid**
Fixing terminology of:
- https://github.com/d2l-ai/d2l-zh/commit/bbf75032a00c21e1d6f4ed293c936469665bac3d#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7
- https://github.com/d2l-ai/d2l-zh/commit/33c9b7f3f9ccde212a2ae4d55dff38a55546804b#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7
- https://github.com/d2l-ai/d2l-zh/commit/e1eb1eb30644ca3787a41ef65df69ffcfcb84b7e#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7
- https://github.com/d2l-ai/d2l-zh/commit/e2d1e2502d4cfe1cdc8bddc34a86c7103f539e41#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7
- https://github.com/d2l-ai/d2l-zh/commit/dbe833e01c86700b41e4ce3743833ca75319e1aa#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7

Excluding (Need to polish by comparing with English version):
- [ ] token: 标记->词元
- [ ] epoch: 迭代周期（周期) -> 轮?

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-897/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-897/

File: TERMINOLOGY.md [hunk 1]
```diff
@@ -166,7 +166,7 @@
 
 批量，batch
 
-平均池化层，average pooling layer
+平均汇聚层，average pooling layer
```
Comment (zhuyuanxiang) [inline TERMINOLOGY.md:169]: Maybe “池化层” is a habituation usage.

**Review by zhuyuanxiang — COMMENTED** [review]
> Fixing terminology of:
> 
> * [bbf7503#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7](https://github.com/d2l-ai/d2l-zh/commit/bbf75032a00c21e1d6f4ed293c936469665bac3d#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7)
> * [33c9b7f#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7](https://github.com/d2l-ai/d2l-zh/commit/33c9b7f3f9ccde212a2ae4d55dff38a55546804b#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7)
> * [e1eb1eb#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7](https://github.com/d2l-ai/d2l-zh/commit/e1eb1eb30644ca3787a41ef65df69ffcfcb84b7e#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7)
> * [e2d1e25#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7](https://github.com/d2l-ai/d2l-zh/commit/e2d1e2502d4cfe1cdc8bddc34a86c7103f539e41#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7)
> * [dbe833e#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7](https://github.com/d2l-ai/d2l-zh/commit/dbe833e01c86700b41e4ce3743833ca75319e1aa#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7)
> 
> Excluding (Need to polish by comparing with English version):
> 
> * [x]  token: 标记->词元
> * [x]  epoch: 迭代周期（周期) -> 轮?

1. 「标记」比「词元」更好，虽然有人曾经建议过「词元」，但是我觉得：token 应该是动词，那么「标记」肯定适合，其表示对某个元素标示一个记号；而「词元」只能表述词的元素化，有的时候不是对词，也有对字母或者词组进行标记
2. 「迭代周期（周期）」比「轮」好，因为「轮」字太简单，在整个句子中容易与其他词混合组词导致整个句子不容易正确分解。「轮」可以用在某些特定场合，可以根据译者的需要调整，大部分时候可以稍稍繁复一些，方便读者理解。

**Comment by goldmermaid** [comment]
> > Fixing terminology of:
> > 
> > * [bbf7503#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7](https://github.com/d2l-ai/d2l-zh/commit/bbf75032a00c21e1d6f4ed293c936469665bac3d#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7)
> > * [33c9b7f#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7](https://github.com/d2l-ai/d2l-zh/commit/33c9b7f3f9ccde212a2ae4d55dff38a55546804b#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7)
> > * [e1eb1eb#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7](https://github.com/d2l-ai/d2l-zh/commit/e1eb1eb30644ca3787a41ef65df69ffcfcb84b7e#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7)
> > * [e2d1e25#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7](https://github.com/d2l-ai/d2l-zh/commit/e2d1e2502d4cfe1cdc8bddc34a86c7103f539e41#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7)
> > * [dbe833e#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7](https://github.com/d2l-ai/d2l-zh/commit/dbe833e01c86700b41e4ce3743833ca75319e1aa#diff-041ec947b33810de1cff8292e421ce34b0968735eda4f655b1325f8fd722e5f7)
> > 
> > Excluding (Need to polish by comparing with English version):
> > 
> > * [ ]   token: 标记->词元
> > * [ ]   epoch: 迭代周期（周期) -> 轮?
> 
> 1. 「标记」比「词元」更好，虽然有人曾经建议过「词元」，但是我觉得：token 应该是动词，那么「标记」肯定适合，其表示对某个元素标示一个记号；而「词元」只能表述词的元素化，有的时候不是对词，也有对字母或者词组进行标记
> 2. 「迭代周期（周期）」比「轮」好，因为「轮」字太简单，在整个句子中容易与其他词混合组词导致整个句子不容易正确分解。「轮」可以用在某些特定场合，可以根据译者的需要调整，大部分时候可以稍稍繁复一些，方便读者理解。

Great comments @zhuyuanxiang ! We will carefully think of your suggestions. (At the mean time, "token" is a noun, "tokenize" is the verb.)

File: TERMINOLOGY.md (hunk 1 above)
Comment (astonzhang) [reply TERMINOLOGY.md:169]: token不是动词。
https://www.lexico.com/en/definition/token

Linguistics
An individual occurrence of a linguistic unit in speech or writing, as contrasted with the type or class of linguistic unit of which it is an instance.

它是自然语言处理的术语。即便是最近的vision Transformer里提及token也明确指出是借用自然语言处理的概念。这样翻译不仅准确而且体现来历。


epoch的翻译我们和《机器学习》、《统计学习方法》、《神经网络与深度学习》几位作者讨论过，没有异议

Comment (goldmermaid) [reply TERMINOLOGY.md:169]: Great explanations! We may need to add these thoughts in "terminology.md", so more customers can better understand it.

Comment (zhuyuanxiang) [reply TERMINOLOGY.md:169]: Thks for your explanations!

Comment (zhuyuanxiang) [reply TERMINOLOGY.md:169]: Could you tell me what is the best translation about the word 'pooling layer'?

**Comment by goldmermaid** [comment]
Merge for the convenience of others' edits. Will fix "token" and "epoch in another PR.

**Pull request merged**